ocs_ci/utility/aws.py

```
@@ -184,6 +184,122 @@ def sts_client(self):
+        Activate or deactivate an IAM access key.
+
+        Args:
+            username (str): IAM username
+            access_key_id (str): The access key ID to update
+            status (str): 'Active' or 'Inactive'
+
+        """
+        self.iam_client.update_access_key(
+            UserName=username, AccessKeyId=access_key_id, Status=status
+        )
+        logger.info(f"Set IAM access key {access_key_id} to {status}")
```

Ruff (0.15.14)

[warning] Logging statement uses f-string

(G004)